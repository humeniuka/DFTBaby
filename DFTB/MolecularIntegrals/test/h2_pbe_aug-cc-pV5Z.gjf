%Chk=H2_pbe_aug-cc-pV5Z.chk
%Mem=5Gb
#P PBEPBE/aug-cc-pV5Z Integral(Grid=UltraFine,Acc2E=14)

hydrogen molecule - experimental geometry

0,1
H	0.0   0.0	     0.0
H	0.0   0.0	     0.741




