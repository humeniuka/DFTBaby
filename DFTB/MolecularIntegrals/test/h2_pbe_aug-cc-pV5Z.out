 Set GAUSS_PDEF to "1".
 Entering Gaussian System, Link 0=g16
 Initial command:
 /local/gaussian-16.A03/g16/l1.exe "/scratch/humeniuka/g16/13042/Gau-22902.inp" -scrdir="/scratch/humeniuka/g16/13042/"
 Entering Link 1 = /local/gaussian-16.A03/g16/l1.exe PID=     22904.
  
 Copyright (c) 1988,1990,1992,1993,1995,1998,2003,2009,2016,
            Gaussian, Inc.  All Rights Reserved.
  
 This is part of the Gaussian(R) 16 program.  It is based on
 the Gaussian(R) 09 system (copyright 2009, Gaussian, Inc.),
 the Gaussian(R) 03 system (copyright 2003, Gaussian, Inc.),
 the Gaussian(R) 98 system (copyright 1998, Gaussian, Inc.),
 the Gaussian(R) 94 system (copyright 1995, Gaussian, Inc.),
 the Gaussian 92(TM) system (copyright 1992, Gaussian, Inc.),
 the Gaussian 90(TM) system (copyright 1990, Gaussian, Inc.),
 the Gaussian 88(TM) system (copyright 1988, Gaussian, Inc.),
 the Gaussian 86(TM) system (copyright 1986, Carnegie Mellon
 University), and the Gaussian 82(TM) system (copyright 1983,
 Carnegie Mellon University). Gaussian is a federally registered
 trademark of Gaussian, Inc.
  
 This software contains proprietary and confidential information,
 including trade secrets, belonging to Gaussian, Inc.
  
 This software is provided under written license and may be
 used, copied, transmitted, or stored only in accord with that
 written license.
  
 The following legend is applicable only to US Government
 contracts under FAR:
  
                    RESTRICTED RIGHTS LEGEND
  
 Use, reproduction and disclosure by the US Government is
 subject to restrictions as set forth in subparagraphs (a)
 and (c) of the Commercial Computer Software - Restricted
 Rights clause in FAR 52.227-19.
  
 Gaussian, Inc.
 340 Quinnipiac St., Bldg. 40, Wallingford CT 06492
  
  
 ---------------------------------------------------------------
 Warning -- This program may not be used in any manner that
 competes with the business of Gaussian, Inc. or will provide
 assistance to any competitor of Gaussian, Inc.  The licensee
 of this program is prohibited from giving any competitor of
 Gaussian, Inc. access to this program.  By using this program,
 the user acknowledges that Gaussian, Inc. is engaged in the
 business of creating and licensing software in the field of
 computational chemistry and represents and warrants to the
 licensee that it is not a competitor of Gaussian, Inc. and that
 it will not use this program in any manner prohibited above.
 ---------------------------------------------------------------
  

 Cite this work as:
 Gaussian 16, Revision A.03,
 M. J. Frisch, G. W. Trucks, H. B. Schlegel, G. E. Scuseria, 
 M. A. Robb, J. R. Cheeseman, G. Scalmani, V. Barone, 
 G. A. Petersson, H. Nakatsuji, X. Li, M. Caricato, A. V. Marenich, 
 J. Bloino, B. G. Janesko, R. Gomperts, B. Mennucci, H. P. Hratchian, 
 J. V. Ortiz, A. F. Izmaylov, J. L. Sonnenberg, D. Williams-Young, 
 F. Ding, F. Lipparini, F. Egidi, J. Goings, B. Peng, A. Petrone, 
 T. Henderson, D. Ranasinghe, V. G. Zakrzewski, J. Gao, N. Rega, 
 G. Zheng, W. Liang, M. Hada, M. Ehara, K. Toyota, R. Fukuda, 
 J. Hasegawa, M. Ishida, T. Nakajima, Y. Honda, O. Kitao, H. Nakai, 
 T. Vreven, K. Throssell, J. A. Montgomery, Jr., J. E. Peralta, 
 F. Ogliaro, M. J. Bearpark, J. J. Heyd, E. N. Brothers, K. N. Kudin, 
 V. N. Staroverov, T. A. Keith, R. Kobayashi, J. Normand, 
 K. Raghavachari, A. P. Rendell, J. C. Burant, S. S. Iyengar, 
 J. Tomasi, M. Cossi, J. M. Millam, M. Klene, C. Adamo, R. Cammi, 
 J. W. Ochterski, R. L. Martin, K. Morokuma, O. Farkas, 
 J. B. Foresman, and D. J. Fox, Gaussian, Inc., Wallingford CT, 2016.
 
 ******************************************
 Gaussian 16:  ES64L-G16RevA.03 25-Dec-2016
                20-Jun-2019 
 ******************************************
 %Chk=H2_pbe_aug-cc-pV5Z.chk
 %Mem=5Gb
 -------------------------------------------------------
 #P PBEPBE/aug-cc-pV5Z Integral(Grid=UltraFine,Acc2E=14)
 -------------------------------------------------------
 1/38=1,172=1/1;
 2/12=2,17=6,18=5,40=1/2;
 3/5=16,6=3,7=10,11=2,25=1,27=14,30=1,74=1009,75=-5/1,2,3;
 4//1;
 5/5=2,38=5,87=14/2;
 6/7=2,8=2,9=2,10=2,28=1,87=14/1;
 99/5=1,9=1/99;
 Leave Link    1 at Thu Jun 20 15:36:03 2019, MaxMem=   671088640 cpu:               0.0 elap:               0.0
 (Enter /local/gaussian-16.A03/g16/l101.exe)
 -----------------------------------------
 hydrogen molecule - experimental geometry
 -----------------------------------------
 Symbolic Z-matrix:
 Charge =  0 Multiplicity = 1
 H                     0.        0.        0. 
 H                     0.        0.        0.741 
 
 ITRead=  0  0
 MicOpt= -1 -1
 NAtoms=      2 NQM=        2 NQMF=       0 NMMI=      0 NMMIF=      0
                NMic=       0 NMicF=      0.
                    Isotopes and Nuclear Properties:
 (Nuclear quadrupole moments (NQMom) in fm**2, nuclear magnetic moments (NMagM)
  in nuclear magnetons)

  Atom         1           2
 IAtWgt=           1           1
 AtmWgt=   1.0078250   1.0078250
 NucSpn=           1           1
 AtZEff=  -0.0000000  -0.0000000
 NQMom=    0.0000000   0.0000000
 NMagM=    2.7928460   2.7928460
 AtZNuc=   1.0000000   1.0000000
 Leave Link  101 at Thu Jun 20 15:36:04 2019, MaxMem=   671088640 cpu:               0.2 elap:               0.2
 (Enter /local/gaussian-16.A03/g16/l202.exe)
                          Input orientation:                          
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          1           0        0.000000    0.000000    0.000000
      2          1           0        0.000000    0.000000    0.741000
 ---------------------------------------------------------------------
 Stoichiometry    H2
 Framework group  D*H[C*(H.H)]
 Deg. of freedom     1
 Full point group                 D*H     NOp   8
 Largest Abelian subgroup         D2H     NOp   8
 Largest concise Abelian subgroup C2      NOp   2
                         Standard orientation:                         
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          1           0        0.000000    0.000000    0.370500
      2          1           0        0.000000    0.000000   -0.370500
 ---------------------------------------------------------------------
 Rotational constants (GHZ):           0.0000000        1826.5250645        1826.5250645
 Leave Link  202 at Thu Jun 20 15:36:04 2019, MaxMem=   671088640 cpu:               0.0 elap:               0.0
 (Enter /local/gaussian-16.A03/g16/l301.exe)
 Standard basis: Aug-CC-pV5Z (5D, 7F)
 Ernie: Thresh=  0.10000D-02 Tol=  0.10000D-05 Strict=F.
 There are    44 symmetry adapted cartesian basis functions of AG  symmetry.
 There are    13 symmetry adapted cartesian basis functions of B1G symmetry.
 There are    24 symmetry adapted cartesian basis functions of B2G symmetry.
 There are    24 symmetry adapted cartesian basis functions of B3G symmetry.
 There are    13 symmetry adapted cartesian basis functions of AU  symmetry.
 There are    44 symmetry adapted cartesian basis functions of B1U symmetry.
 There are    24 symmetry adapted cartesian basis functions of B2U symmetry.
 There are    24 symmetry adapted cartesian basis functions of B3U symmetry.
 There are    31 symmetry adapted basis functions of AG  symmetry.
 There are    11 symmetry adapted basis functions of B1G symmetry.
 There are    19 symmetry adapted basis functions of B2G symmetry.
 There are    19 symmetry adapted basis functions of B3G symmetry.
 There are    11 symmetry adapted basis functions of AU  symmetry.
 There are    31 symmetry adapted basis functions of B1U symmetry.
 There are    19 symmetry adapted basis functions of B2U symmetry.
 There are    19 symmetry adapted basis functions of B3U symmetry.
   160 basis functions,   216 primitive gaussians,   210 cartesian basis functions
     1 alpha electrons        1 beta electrons
       nuclear repulsion energy         0.7141392860 Hartrees.
 IExCor= 1009 DFT=T Ex=PBE Corr=PBE ExCW=0 ScaHFX=  0.000000
 ScaDFX=  1.000000  1.000000  1.000000  1.000000 ScalE2=  1.000000  1.000000
 IRadAn=      5 IRanWt=     -1 IRanGd=            0 ICorTp=0 IEmpDi=  4
 NAtoms=    2 NActive=    2 NUniq=    1 SFac= 4.00D+00 NAtFMM=   60 NAOKFM=F Big=F
 Integral buffers will be    131072 words long.
 Raffenetti 2 integral format.
 Two-electron integral symmetry is turned on.
 Leave Link  301 at Thu Jun 20 15:36:04 2019, MaxMem=   671088640 cpu:               0.1 elap:               0.1
 (Enter /local/gaussian-16.A03/g16/l302.exe)
 NPDir=0 NMtPBC=     1 NCelOv=     1 NCel=       1 NClECP=     1 NCelD=      1
         NCelK=      1 NCelE2=     1 NClLst=     1 CellRange=     0.0.
 One-electron integrals computed using PRISM.
 One-electron integral symmetry used in STVInt
 NBasis=   160 RedAO= T EigKep=  5.63D-06  NBF=    31    11    19    19    11    31    19    19
 NBsUse=   160 1.00D-06 EigRej= -1.00D+00 NBFU=    31    11    19    19    11    31    19    19
 Precomputing XC quadrature grid using
 IXCGrd= 4 IRadAn=           5 IRanWt=          -1 IRanGd=           0 AccXCQ= 1.00D-14.
 Generated NRdTot=       0 NPtTot=           0 NUsed=           0 NTot=          32
 NSgBfM=   210   210   210   210   210 MxSgAt=     2 MxSgA2=     2.
 Leave Link  302 at Thu Jun 20 15:36:04 2019, MaxMem=   671088640 cpu:               0.3 elap:               0.3
 (Enter /local/gaussian-16.A03/g16/l303.exe)
 DipDrv:  MaxL=1.
 Leave Link  303 at Thu Jun 20 15:36:04 2019, MaxMem=   671088640 cpu:               0.1 elap:               0.1
 (Enter /local/gaussian-16.A03/g16/l401.exe)
 ExpMin= 2.07D-02 ExpMax= 4.02D+02 ExpMxC= 6.02D+01 IAcc=3 IRadAn=         5 AccDes= 0.00D+00
 Harris functional with IExCor= 1009 and IRadAn=       5 diagonalized for initial guess.
 HarFok:  IExCor= 1009 AccDes= 0.00D+00 IRadAn=         5 IDoV= 1 UseB2=F ITyADJ=14
 ICtDFT=  3500011 ScaDFX=  1.000000  1.000000  1.000000  1.000000
 FoFCou: FMM=F IPFlag=           0 FMFlag=      100000 FMFlg1=           0
         NFxFlg=           0 DoJE=T BraDBF=F KetDBF=T FulRan=T
         wScrn=  0.000000 ICntrl=       500 IOpCl=  0 I1Cent=   200000004 NGrid=           0
         NMat0=    1 NMatS0=      1 NMatT0=    0 NMatD0=    1 NMtDS0=    0 NMtDT0=    0
 Petite list used in FoFCou.
 Harris En= -1.18230122263428    
 JPrj=0 DoOrth=F DoCkMO=F.
 Initial guess orbital symmetries:
       Occupied  (SGG)
       Virtual   (SGU) (SGG) (PIU) (PIU) (SGU) (SGG) (PIG) (PIG)
                 (SGG) (SGU) (SGU) (DLTG) (DLTG) (PIU) (PIU) (SGG)
                 (PIG) (PIG) (DLTU) (DLTU) (PIU) (PIU) (SGU) (SGG)
                 (SGU) (PHIU) (PHIU) (PIG) (PIG) (SGG) (PIG) (PIG)
                 (PIU) (PIU) (PHIG) (PHIG) (DLTU) (DLTU) (SGG)
                 (DLTG) (DLTG) (SGU) (DLTG) (DLTG) (SGU) (PIU)
                 (PIU) (SGG) (PIG) (PIG) (SGU) (SGG) (PIU) (PIU)
                 (DLTU) (DLTU) (?A) (?A) (PIG) (PIG) (DLTU) (DLTU)
                 (SGU) (PIU) (PIU) (PHIG) (PHIG) (PHIU) (PHIU)
                 (DLTG) (DLTG) (?B) (?B) (SGG) (SGU) (PIG) (PIG)
                 (SGG) (PHIU) (PHIU) (SGU) (DLTG) (DLTG) (PIU)
                 (PIU) (PIG) (PIG) (SGG) (DLTG) (DLTG) (PHIG) (PHIG)
                 (DLTU) (DLTU) (PIU) (PIU) (SGG) (SGU) (PIG) (PIG)
                 (SGU) (DLTU) (DLTU) (PIU) (PIU) (SGU) (SGG) (PIG)
                 (PIG) (PHIU) (PHIU) (DLTG) (DLTG) (PIU) (PIU)
                 (?A) (?A) (SGG) (PHIU) (PHIU) (PHIG) (PHIG) (?B)
                 (?B) (DLTG) (DLTG) (PIG) (PIG) (SGG) (SGU) (DLTU)
                 (DLTU) (PIU) (PIU) (DLTU) (DLTU) (SGG) (DLTG)
                 (DLTG) (PIG) (PIG) (PHIG) (PHIG) (SGU) (SGU) (PIU)
                 (PIU) (SGG) (PIG) (PIG) (PIU) (PIU) (DLTU) (DLTU)
                 (SGU) (SGG) (PIG) (PIG) (SGU)
 The electronic state of the initial guess is 1-SGG.
 Leave Link  401 at Thu Jun 20 15:36:04 2019, MaxMem=   671088640 cpu:               0.4 elap:               0.4
 (Enter /local/gaussian-16.A03/g16/l502.exe)
 Keep J ints in memory in symmetry-blocked form, NReq=93466019.
 FoFCou: FMM=F IPFlag=           0 FMFlag=           0 FMFlg1=           0
         NFxFlg=           0 DoJE=F BraDBF=F KetDBF=F FulRan=T
         wScrn=  0.000000 ICntrl=       600 IOpCl=  0 I1Cent=           0 NGrid=           0
         NMat0=    1 NMatS0=  12880 NMatT0=    0 NMatD0=    1 NMtDS0=    0 NMtDT0=    0
 Petite list used in FoFCou.
 Closed shell SCF:
 Using DIIS extrapolation, IDIIS=  1040.
 NGot=   671088640 LenX=   660486016 LenY=   660441475
 Requested convergence on RMS density matrix=1.00D-08 within 128 cycles.
 Requested convergence on MAX density matrix=1.00D-06.
 Requested convergence on             energy=1.00D-06.
 No special actions if energy rises.

 Cycle   1  Pass 1  IDiag  1:
 E= -1.16181131598324    
 DIIS: error= 2.37D-02 at cycle   1 NSaved=   1.
 NSaved= 1 IEnMin= 1 EnMin= -1.16181131598324     IErMin= 1 ErrMin= 2.37D-02
 ErrMax= 2.37D-02  0.00D+00 EMaxC= 1.00D-01 BMatC= 2.98D-03 BMatP= 2.98D-03
 IDIUse=3 WtCom= 7.63D-01 WtEn= 2.37D-01
 Coeff-Com:  0.100D+01
 Coeff-En:   0.100D+01
 Coeff:      0.100D+01
 Gap=     0.368 Goal=   None    Shift=    0.000
 GapD=    0.368 DampG=1.000 DampE=0.500 DampFc=0.5000 IDamp=-1.
 Damping current iteration by 5.00D-01
 RMSDP=2.41D-04 MaxDP=9.70D-03              OVMax= 7.17D-02

 Cycle   2  Pass 1  IDiag  1:
 E= -1.16450640154234     Delta-E=       -0.002695085559 Rises=F Damp=T
 DIIS: error= 7.62D-03 at cycle   2 NSaved=   2.
 NSaved= 2 IEnMin= 2 EnMin= -1.16450640154234     IErMin= 2 ErrMin= 7.62D-03
 ErrMax= 7.62D-03  0.00D+00 EMaxC= 1.00D-01 BMatC= 3.13D-04 BMatP= 2.98D-03
 IDIUse=3 WtCom= 9.24D-01 WtEn= 7.62D-02
 Coeff-Com: -0.479D+00 0.148D+01
 Coeff-En:   0.000D+00 0.100D+01
 Coeff:     -0.442D+00 0.144D+01
 Gap=     0.390 Goal=   None    Shift=    0.000
 RMSDP=5.58D-05 MaxDP=1.86D-03 DE=-2.70D-03 OVMax= 1.88D-02

 Cycle   3  Pass 1  IDiag  1:
 E= -1.16667399747280     Delta-E=       -0.002167595930 Rises=F Damp=F
 DIIS: error= 7.11D-05 at cycle   3 NSaved=   3.
 NSaved= 3 IEnMin= 3 EnMin= -1.16667399747280     IErMin= 3 ErrMin= 7.11D-05
 ErrMax= 7.11D-05  0.00D+00 EMaxC= 1.00D-01 BMatC= 3.73D-08 BMatP= 3.13D-04
 IDIUse=1 WtCom= 1.00D+00 WtEn= 0.00D+00
 Coeff-Com:  0.250D-01-0.687D-01 0.104D+01
 Coeff:      0.250D-01-0.687D-01 0.104D+01
 Gap=     0.391 Goal=   None    Shift=    0.000
 RMSDP=2.06D-06 MaxDP=8.38D-05 DE=-2.17D-03 OVMax= 3.20D-04

 Cycle   4  Pass 1  IDiag  1:
 E= -1.16667410288062     Delta-E=       -0.000000105408 Rises=F Damp=F
 DIIS: error= 5.18D-06 at cycle   4 NSaved=   4.
 NSaved= 4 IEnMin= 4 EnMin= -1.16667410288062     IErMin= 4 ErrMin= 5.18D-06
 ErrMax= 5.18D-06  0.00D+00 EMaxC= 1.00D-01 BMatC= 1.63D-10 BMatP= 3.73D-08
 IDIUse=1 WtCom= 1.00D+00 WtEn= 0.00D+00
 Coeff-Com:  0.110D-02-0.388D-02 0.224D-01 0.980D+00
 Coeff:      0.110D-02-0.388D-02 0.224D-01 0.980D+00
 Gap=     0.391 Goal=   None    Shift=    0.000
 RMSDP=7.86D-08 MaxDP=2.41D-06 DE=-1.05D-07 OVMax= 1.61D-05

 Cycle   5  Pass 1  IDiag  1:
 E= -1.16667410321547     Delta-E=       -0.000000000335 Rises=F Damp=F
 DIIS: error= 4.52D-07 at cycle   5 NSaved=   5.
 NSaved= 5 IEnMin= 5 EnMin= -1.16667410321547     IErMin= 5 ErrMin= 4.52D-07
 ErrMax= 4.52D-07  0.00D+00 EMaxC= 1.00D-01 BMatC= 1.21D-12 BMatP= 1.63D-10
 IDIUse=1 WtCom= 1.00D+00 WtEn= 0.00D+00
 Coeff-Com:  0.725D-04-0.131D-03 0.119D-02-0.318D-01 0.103D+01
 Coeff:      0.725D-04-0.131D-03 0.119D-02-0.318D-01 0.103D+01
 Gap=     0.391 Goal=   None    Shift=    0.000
 RMSDP=5.69D-09 MaxDP=3.04D-07 DE=-3.35D-10 OVMax= 7.76D-07

 SCF Done:  E(RPBE-PBE) =  -1.16667410322     A.U. after    5 cycles
            NFock=  5  Conv=0.57D-08     -V/T= 2.0227
 KE= 1.140805644792D+00 PE=-3.644577087112D+00 EE= 6.229580531121D-01
 Leave Link  502 at Thu Jun 20 15:36:15 2019, MaxMem=   671088640 cpu:              10.4 elap:              10.4
 (Enter /local/gaussian-16.A03/g16/l601.exe)
 Copying SCF densities to generalized density rwf, IOpCl= 0 IROHF=0.

 **********************************************************************

            Population analysis using the SCF density.

 **********************************************************************

 Orbital symmetries:
       Occupied  (SGG)
       Virtual   (SGU) (SGG) (PIU) (PIU) (SGU) (SGG) (PIG) (PIG)
                 (SGG) (SGU) (SGU) (DLTG) (DLTG) (PIU) (PIU) (SGG)
                 (PIG) (PIG) (DLTU) (DLTU) (PIU) (PIU) (SGU) (SGG)
                 (SGU) (PHIU) (PHIU) (PIG) (PIG) (SGG) (PIG) (PIG)
                 (PHIG) (PHIG) (PIU) (PIU) (DLTU) (DLTU) (SGG)
                 (DLTG) (DLTG) (SGU) (DLTG) (DLTG) (SGU) (PIU)
                 (PIU) (SGG) (PIG) (PIG) (SGU) (SGG) (PIU) (PIU)
                 (DLTU) (DLTU) (?A) (?A) (PIG) (PIG) (DLTU) (DLTU)
                 (SGU) (PIU) (PIU) (PHIG) (PHIG) (PHIU) (PHIU)
                 (DLTG) (DLTG) (?B) (?B) (SGG) (SGU) (PIG) (PIG)
                 (SGG) (PHIU) (PHIU) (SGU) (DLTG) (DLTG) (PIU)
                 (PIU) (PIG) (PIG) (SGG) (DLTG) (DLTG) (PHIG) (PHIG)
                 (DLTU) (DLTU) (PIU) (PIU) (SGG) (SGU) (PIG) (PIG)
                 (SGU) (DLTU) (DLTU) (PIU) (PIU) (SGU) (SGG) (PIG)
                 (PIG) (PHIU) (PHIU) (DLTG) (DLTG) (PIU) (PIU)
                 (?A) (?A) (SGG) (PHIU) (PHIU) (PHIG) (PHIG) (?B)
                 (?B) (DLTG) (DLTG) (PIG) (PIG) (SGG) (SGU) (DLTU)
                 (DLTU) (PIU) (PIU) (DLTU) (DLTU) (SGG) (DLTG)
                 (DLTG) (PIG) (PIG) (PHIG) (PHIG) (SGU) (SGU) (PIU)
                 (PIU) (SGG) (PIG) (PIG) (PIU) (PIU) (DLTU) (DLTU)
                 (SGU) (SGG) (PIG) (PIG) (SGU)
 The electronic state is 1-SGG.
 Alpha  occ. eigenvalues --   -0.38153
 Alpha virt. eigenvalues --    0.00907   0.01338   0.07204   0.07204   0.07638
 Alpha virt. eigenvalues --    0.14424   0.15808   0.15808   0.18318   0.24573
 Alpha virt. eigenvalues --    0.28259   0.38090   0.38090   0.44756   0.44756
 Alpha virt. eigenvalues --    0.46328   0.46463   0.46463   0.47639   0.47639
 Alpha virt. eigenvalues --    0.47946   0.47946   0.62533   0.70395   0.74429
 Alpha virt. eigenvalues --    1.00245   1.00245   1.03930   1.03930   1.07294
 Alpha virt. eigenvalues --    1.19943   1.19943   1.20803   1.20803   1.20946
 Alpha virt. eigenvalues --    1.20946   1.22126   1.22126   1.22412   1.22789
 Alpha virt. eigenvalues --    1.22789   1.28958   1.52676   1.52676   1.53604
 Alpha virt. eigenvalues --    1.56768   1.56768   2.00573   2.13022   2.13022
 Alpha virt. eigenvalues --    2.20587   2.38246   2.51556   2.51556   2.56746
 Alpha virt. eigenvalues --    2.56746   2.63087   2.63087   2.84230   2.84230
 Alpha virt. eigenvalues --    2.92222   2.92222   2.93963   2.95771   2.95771
 Alpha virt. eigenvalues --    3.01446   3.01446   3.09640   3.09640   3.16258
 Alpha virt. eigenvalues --    3.16258   3.16812   3.16812   3.26341   3.28421
 Alpha virt. eigenvalues --    3.74522   3.74522   3.76452   3.76915   3.76915
 Alpha virt. eigenvalues --    4.17010   4.21418   4.21418   4.57988   4.57988
 Alpha virt. eigenvalues --    4.88850   4.88850   4.95739   5.30084   5.30084
 Alpha virt. eigenvalues --    5.37445   5.37445   5.46768   5.46768   5.52316
 Alpha virt. eigenvalues --    5.52316   5.57824   6.46682   6.49745   6.49745
 Alpha virt. eigenvalues --    6.69350   6.76414   6.76414   7.02841   7.02841
 Alpha virt. eigenvalues --    8.09094   8.35341   8.68773   8.68773  11.61425
 Alpha virt. eigenvalues --   11.61425  11.67526  11.67526  12.00126  12.00126
 Alpha virt. eigenvalues --   12.31100  12.31100  12.74887  12.78263  12.78263
 Alpha virt. eigenvalues --   13.18483  13.18483  13.28871  13.28871  13.39048
 Alpha virt. eigenvalues --   13.39048  13.62872  13.62872  13.64851  13.74078
 Alpha virt. eigenvalues --   13.77689  13.77689  13.92799  13.92799  14.45587
 Alpha virt. eigenvalues --   14.45587  14.47925  14.58557  14.58557  14.96568
 Alpha virt. eigenvalues --   14.96568  15.03703  15.03703  15.35620  16.63964
 Alpha virt. eigenvalues --   16.73233  16.73233  16.87077  17.22478  17.22478
 Alpha virt. eigenvalues --   18.26673  18.26673  18.49769  18.49769  18.91806
 Alpha virt. eigenvalues --   22.51900  23.56741  23.56741  28.32154
          Condensed to atoms (all electrons):
               1          2
     1  H    0.572211   0.427789
     2  H    0.427789   0.572211
 Mulliken charges:
               1
     1  H   -0.000000
     2  H   -0.000000
 Sum of Mulliken charges =  -0.00000
 Mulliken charges with hydrogens summed into heavy atoms:
               1
 Electronic spatial extent (au):  <R**2>=              5.2633
 Charge=             -0.0000 electrons
 Dipole moment (field-independent basis, Debye):
    X=             -0.0000    Y=             -0.0000    Z=              0.0000  Tot=              0.0000
 Quadrupole moment (field-independent basis, Debye-Ang):
   XX=             -2.1133   YY=             -2.1133   ZZ=             -1.5340
   XY=             -0.0000   XZ=             -0.0000   YZ=              0.0000
 Traceless Quadrupole moment (field-independent basis, Debye-Ang):
   XX=             -0.1931   YY=             -0.1931   ZZ=              0.3862
   XY=             -0.0000   XZ=             -0.0000   YZ=              0.0000
 Octapole moment (field-independent basis, Debye-Ang**2):
  XXX=             -0.0000  YYY=             -0.0000  ZZZ=              0.0000  XYY=             -0.0000
  XXY=              0.0000  XXZ=              0.0000  XZZ=             -0.0000  YZZ=             -0.0000
  YYZ=              0.0000  XYZ=              0.0000
 Hexadecapole moment (field-independent basis, Debye-Ang**3):
 XXXX=             -2.2342 YYYY=             -2.2342 ZZZZ=             -3.0202 XXXY=             -0.0000
 XXXZ=              0.0000 YYYX=             -0.0000 YYYZ=              0.0000 ZZZX=              0.0000
 ZZZY=              0.0000 XXYY=             -0.7447 XXZZ=             -0.8921 YYZZ=             -0.8921
 XXYZ=              0.0000 YYXZ=              0.0000 ZZXY=             -0.0000
 N-N= 7.141392859919D-01 E-N=-3.644577625066D+00  KE= 1.140805644792D+00
 Symmetry AG   KE= 1.140805644792D+00
 Symmetry B1G  KE= 5.129485170889D-52
 Symmetry B2G  KE= 1.355623025836D-29
 Symmetry B3G  KE= 1.355623025840D-29
 Symmetry AU   KE=-8.047487133135D-55
 Symmetry B1U  KE=-1.003828482430D-21
 Symmetry B2U  KE= 6.643503468074D-32
 Symmetry B3U  KE= 6.643503468073D-32
 No NMR shielding tensors so no spin-rotation constants.
 Leave Link  601 at Thu Jun 20 15:36:15 2019, MaxMem=   671088640 cpu:               0.3 elap:               0.3
 (Enter /local/gaussian-16.A03/g16/l9999.exe)
 1\1\GINC-WUX18\SP\RPBEPBE\Aug-CC-pV5Z\H2\HUMENIUKA\20-Jun-2019\0\\#P P
 BEPBE/aug-cc-pV5Z Integral(Grid=UltraFine,Acc2E=14)\\hydrogen molecule
  - experimental geometry\\0,1\H,0,0.,0.,0.\H,0,0.,0.,0.741\\Version=ES
 64L-G16RevA.03\State=1-SGG\HF=-1.1666741\RMSD=5.687e-09\Dipole=0.,0.,0
 .\Quadrupole=-0.1435554,-0.1435554,0.2871109,0.,0.,0.\PG=D*H [C*(H1.H1
 )]\\@


 REPARTEE - WHAT YOU THINK OF AFTER YOU BECOME A DEPARTEE.
 Job cpu time:       0 days  0 hours  0 minutes 11.9 seconds.
 Elapsed time:       0 days  0 hours  0 minutes 11.8 seconds.
 File lengths (MBytes):  RWF=     16 Int=      0 D2E=      0 Chk=      3 Scr=      1
 Normal termination of Gaussian 16 at Thu Jun 20 15:36:15 2019.
